tests/percy/index.js

```diff
@@ -17,11 +18,18 @@ PercyScript.run(async (page, percySnapshot) => {
 
   const standardPageNavigator = new StandardPageNavigator(page, `http://localhost:${PORT}`);
   const iframePageNavigator = new IframePageNavigator(page, `http://localhost:${PORT}`, 'iframe_test');
+
   const standardCamera = new Camera(percySnapshot);
   const iframeCamera = new Camera(percySnapshot, true);
 
   await (new Photographer(standardPageNavigator, standardCamera).captureSnapshots());
   await (new Photographer(iframePageNavigator, iframeCamera).captureSnapshots());
 
+  standardPageNavigator.setCurrentLocale('es');
```

Comment:
what do you think about just using `setLocale`?

Reply:
Since navigator has a defaultLocale field to help out with the url logic, and I thought using `setLocale` may be less clear whether it's setting the default or current locale for the site.

Reply:
oh, I think `setCurrentLocale` makes sense then